packages/lit-dev-content/site/css/tutorial.css

```diff
@@ -286,6 +286,20 @@ resize-bar[active] {
   display: none;
 }
 
+.checkButton.checking svg {
+  animation: spin 5s linear infinite;
+}
+
+@keyframes spin {
```

Comment:
The loading state is so fast that this is never shown. May almost be worth adding a slow-down to the checking state to handle this case.

Reply:
Like an artificial slowdown?

Reply:
oh I misread. Yeah, i put it in and tested that speed just for this reason

Reply:
Yeah. One second is perceived as instant, and would give some time to see the spinner. Right now it flashes by - at least in the current example.

Reply:
I added a delay to the second step of this tutorial so you can demo it